### ISSUE TITLE
download (Does not work) W3C Web Content Accessibility Guidelines (WCAG) 2.1 Conformance Statement for Power Apps portals

### ISSUE THREAD

**Issue by losi127** [MicrosoftDocs/powerapps-docs#2929]
/// URL not works -> https://celaaatprod.blob.core.windows.net/public/2ada9a2e-66df-430a-94f7-559b51e07f4e/a1a8f896-f531-4e0b-b5b3-0ccac477d1e5/Power%20Apps%20portals.Web.WCAG.docx (Blob error message)
---
![BLOB](https://user-images.githubusercontent.com/32731795/146771492-5da67cc0-cb6d-4d98-8ffb-c40f533a310b.jpg)

This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Error>
<Code>ResourceNotFound</Code>
<Message>The specified resource does not exist. RequestId:ef23a4c1-c01e-0064-1ba2-f571be000000 Time:2021-12-20T13:03:58.4586088Z</Message>
</Error>

#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b8e40af7-ba15-42f8-b9e1-3d1c5ae73285
* Version Independent ID: d63fa884-aa15-e95d-f44b-8ce6eabf7142
* Content: [Accessibility in Power Apps portals - Power Apps](https://docs.microsoft.com/en-us/powerapps/maker/portals/admin/accessibility)
* Content Source: [powerapps-docs/maker/portals/admin/accessibility.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/maker/portals/admin/accessibility.md)
* Service: **powerapps**
* Sub-service: **portals**
* GitHub Login: @neerajnandwana-msft
* Microsoft Alias: **nenandw**

**Comment by nickdoelman** [comment]
Thank you for bringing this issue to our attention. The topic has been updated with links to the **Microsoft Accessibility Conformance Reports** to download these documents.
